pytups/superdict.py

```diff
@@ -454,16 +460,21 @@ def kapply(self, func: Callable, *args, **kwargs) -> "SuperDict":
         """
         return SuperDict({k: func(k, *args, **kwargs) for k in self})
 
-    def sapply(self, func: Callable, other: dict, *args, **kwargs) -> "SuperDict":
+    def sapply(
+        self, func: Callable, other: Union[dict, int, float], *args, **kwargs
+    ) -> "SuperDict":
         """
         Applies function to both dictionaries.
         Using keys of the self.
         It's like applying a function over the left join.
 
         :param callable func: function to apply.
-        :param dict other: dictionary to apply function to
+        :param Union[dict, int, float] other: either ar int or float to perform the operation over or another dictionary
```

Comment:
typo: "ar" should be "an"
better: 
`either an int, a float or another dictionary to perform the operation over`